contracts/Staking.sol

```diff
@@ -437,6 +440,8 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
         ISQToken(SQToken).burn(burnAmount);
         IERC20(SQToken).safeTransfer(msg.sender, availableAmount);
 
+        stakedAmount[msg.sender] -= amount;
```

Comment:
This should in `_startUnbond` other than`_withdrawARequest`, be consistent with `totalStakingAmount`

Reply:
During the locked period, the token is not `staked` as it is removed from total staking.

Reply:
Can we change `stakedAmount` to `lockedAmount`, so lockedAmount include `stakedAmount` + `amount in locked period`. @ianhe8x @neo-sun

Reply:
yeah, lockedAmount will be better.